Konnect Explorer Reference and LLM Reference (#523)
Kong/developer.konghq.com: app/konnect/advanced-analytics/llm-reporting.md

| Prompt Tokens         | Count         | Prompt tokens are the tokens input into the model.            |
| Total Tokens          | Count         | Sum of all tokens used in a single request to the model. It includes both the tokens in the input (prompt) and the tokens generated by the model (completion).  |
| Time per Tokens       | Number  | Average time in milliseconds to generate a token. Calculated as LLM latency divided by the number of tokens.                                                       |
| Costs                 | Cost  | Represents the resulting costs for a request. Final costs = (total number of prompt tokens × input cost per token) + (total number of completion tokens × output cost per token) + (total number of prompt tokens × embedding cost per token). |
| Response Model        | String        | Represents which AI model was used to process the prompt by the AI provider.   |
| Request Model         | String        | Represents which AI model was used to process the prompt. |
| Provider Name         | String        | Represents which AI provider was used to process the prompt.   |

> | Plugin ID             | String        | Represents the UUID of the plugin configuration.          |

Reviewer: ```suggestion
| Plugin ID             | String        | Represents the UUID of the [plugin](/plugins/) configuration.          |
```
Or maybe this should be the link to AI plugins? I think this means just AI plugins, but I'm not sure.

Author: Removed the mention of AI

| LLM Latency           | Latency  | Total time taken to receive a full response after a request sent from Kong (LLM latency + connection time).             |
| Embeddings Latency    | Latency  | Time taken to generate the vector for the prompt string.  |
| Fetch Latency         | Latency | Total time taken to return a cache.                          |
| Cache Status          | String        | Shows if the response comes directly from the upstream or not. Possible values: `hit` or `Miss`.                                          |
| Embeddings Model      | String        | AI providers may have multiple embedding models. This represents the model used for the embeddings.                      |
| Embeddings Provider   | String        | Provider used for generating embeddings.                                                            |
| Embeddings Token      | Count         | Tokens input into the model for embeddings.  |
| Embeddings Cost       | Cost  | Cost of caching.  |

> | Cost Savings          | Cost  | Cost savings from cache.            |

Reviewer: Nice to have: I'd like more info about how the the cost savings from the cache is calculated.

## Time intervals

The time frame selector controls the time frame of data visualized, which indirectly controls the
granularity of the data. For example, the “5M” selection displays five minutes in
one-second resolution data, while longer time frames display minute, hour, or days resolution data.

All time interval presets are **relative**. 
For custom reports, you can also choose a **custom** date range.

* **Relative** time frames are dynamic and the report captures a snapshot of data
relative to when a user views the report.
* **Custom** time frames are static and the report captures a snapshot of data
during the specified time frame. You can see the exact range below
the time frame selector. For example:

    ```
    Jan 26, 2023 12:00 AM - Feb 01, 2023 12:00 AM (PST)
    ```

> 

Reviewer: ```suggestion
The following table describes the time intervals you can select:

```

> Interval | Description  
> ---------|-------------
> Last 15 minutes | Data is aggregated in one minute increments.
> Last hour| Data is aggregated in one minute increments.
> Last six hours | Data is aggregated in one minute increments.
> Last 12 hours| Data is aggregated in one hour increments.
> Last 24 hours| Data is aggregated in one hour increments.
> Last seven days | Data is aggregated in one hour increments.
> Last 30 days | Data is aggregated in daily increments.
> Current week | Data is aggregated in one hour increments. Logs any traffic in the current calendar week. 
> Current month | Data is aggregated in one hour increments. Logs any traffic in the current calendar month. 
> Previous week | Data is aggregated in one hour increments. Logs any traffic in the previous calendar week.
> Previous month | Data is aggregated in daily increments. Logs any traffic in the previous calendar month. 

Reviewer: ```suggestion
|Interval | Aggregation increment frequency | Notes |  
|---------|-------------|--------------|
| Last 15 minutes | 1 minute |  |
| Last hour | 1 minute  |  |
| Last six hours |  1 minute  |  |
| Last 12 hours |  1 hour  |  |
| Last 24 hours |  1 hour  |  |
| Last seven days |  1 hour  |  |
| Last 30 days |  Daily  |  |
| Current week | 1 hour   |  Logs any traffic in the current calendar week.  |
| Current month |  1 hour  |  Logs any traffic in the current calendar month.  |
| Previous week |  1 hour  | Logs any traffic in the previous calendar week. |
| Previous month |  Daily  |  Logs any traffic in the previous calendar month.  |
```
I wanted to make it easier to scan the increments, but it might look odd to only have notes for some entries.
